docs/doxygen/migration/migration.txt

```diff
@@ -0,0 +1,7 @@
+/**
+@page migration_guide Migration Guide
+@brief Migration guide for v3 libraries to the 202009.00 release.
```

Comment:
```suggestion
@brief Migration guide for v3 library to the 202009.00 release.
```

Reply:
I think I'll keep it since both shadow and mqtt are here